.circleci/config.yml

```diff
@@ -105,7 +105,7 @@ jobs:
 
   tidy-go:
```

Comment:
Could you also update the format-go job above? There are currently 4 places where `cimg/go:1.17.4` is used.